core/services/eth/log_broadcaster.go

```diff
@@ -216,14 +209,63 @@ ResubscribeLoop:
 		if err != nil {
 			logger.Error(err)
 			b.notifyDisconnect()
-			continue ResubscribeLoop
+			continue
 		} else if !shouldResubscribe {
 			b.notifyDisconnect()
 			return
 		}
 	}
 }
 
+func (b *logBroadcaster) backfillLogs() (chBackfilledLogs chan eth.Log, abort bool) {
+	if len(b.listeners) == 0 {
+		ch := make(chan eth.Log)
+		close(ch)
+		return ch, false
+	}
+
+	abort = utils.RetryWithBackoff(b.chStop, "backfilling logs", func() error {
+		latestBlock, err := b.ethClient.GetLatestBlock()
```

Comment:
If a node was down for a while and restarted we would want to get logs further back than 10 blocks ago, right? Maybe we can pass this in from somewhere else.

Reply:
Nevermind, I see now.